autoscale/src/main/java/com/sequenceiq/periscope/domain/ScalingActivity.java

```diff
@@ -0,0 +1,131 @@
+package com.sequenceiq.periscope.domain;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import com.sequenceiq.periscope.api.model.ActivityStatus;
+
+@Entity
+@Table(name = "scaling_activity")
+public class ScalingActivity implements Clustered {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "scaling_activity_generator")
+    @SequenceGenerator(name  = "scaling_activity_generator", sequenceName = "scaling_activity_id_seq", allocationSize = 1)
+    private Long id;
+
+    @Column(name = "activity_crn")
+    private String activityCrn;
+
+    @Column(name = "flow_id")
+    private String flowId;
+
+    @Column(name = "start_time")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date startTime;
+
+    @Column(name = "end_time")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date endTime;
+
+    @Column(name = "activity_reason")
+    private String scalingActivityReason;
+
+    @Column(name = "activity_status")
+    @Enumerated(EnumType.STRING)
```

Comment:
Sanity check: I could find only a few references to `@Enumerated` in CB. Most enum entity properties are mapped using `@Convert(converter = XYZ.class)` where `XYZ` is a specific `DefaultEnumConverter` subclass. Is `@Enumerated` really the desired approach here?

Reply:
I used `@Enumerated` because it was followed in another periscope entity.
https://github.com/hortonworks/cloudbreak/blob/6b6cb4ec1dc0f8e203c27801e073aaba7b47c1f7/autoscale/src/main/java/com/sequenceiq/periscope/domain/Cluster.java#L50-L51

I can change this to use a converter if that is a better approach

Reply:
Updated this to use a converter instead